src/screens/DataClearFilters.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Data, DataClearFilters } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import { ComponentDoc } from '../components/Doc';
+
+const DataClearFiltersPage = () => (
+  <Page>
+    <ComponentDoc
+      name="DataClearFilters"
+      stable
+      availableAt={[
```

Comment:
Should we also link the storybook example here?
https://storybook.grommet.io/?path=/story/data-dataclearfilters-simple--simple

Reply:
Fixed.